lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Boring
+  module DatabaseCleaner
+    module ActiveRecord
+      module Rspec
+        class InstallGenerator < Rails::Generators::Base
+          desc 'Adds database_cleaner-active_record gem to the application for cleaning ActiveRecord databases using different strategies'
+
+          TEST_FRAMEWORK = 'rspec-rails'
+
+          def add_database_cleaner_active_record_gem
+            say 'Adding database_cleaner-active_record gem', :green
+
+            Bundler.with_unbundled_env do
+              run "bundle add database_cleaner-active_record --group='test'"
+            end
+          end
+
+          def configure_database_cleaner
+            verify_presence_of_rspec_gem
```

Comment:
Let's remove this from here, we need to verify the presence as the first step before even adding the database cleaner gem to the project.

Reply:
You can see this type of verifications in other open PRs like rswag: https://github.com/abhaynikam/boring_generators/pull/83/files